pages/api/polling/relayer-balance.ts

```diff
@@ -0,0 +1,32 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import withApiHandler from 'modules/app/api/withApiHandler';
+import logger from 'lib/logger';
+import { isSupportedNetwork } from 'modules/web3/helpers/networks';
+import { getMessageFromCode, ERROR_CODES } from 'eth-rpc-errors';
+import { getRelayerBalance } from 'modules/polling/helpers/getRelayerBalance';
+import { DEFAULT_NETWORK } from 'modules/web3/constants/networks';
+import invariant from 'tiny-invariant';
+
+export default withApiHandler(
+  async (req: NextApiRequest, res: NextApiResponse) => {
+    try {
+      const network = (req.query.network as string) || DEFAULT_NETWORK.network;
+      invariant(isSupportedNetwork(network), `unsupported network ${network}`);
+
+      const balance = await getRelayerBalance(network);
```

Comment:
we could just do a regular `provider.getBalance` in the frontend with the hardcoded address too. Maybe that would be easier but it's rather easy to get the address from the relayer signer too. 🤷

Reply:
is really nice what you did with the relayer api, is more reliable

Reply:
Yeah this sounds good too!